src/com/google/foam/demos/olympics/Controller.js

```diff
@@ -30,8 +30,13 @@ foam.CLASS({
     'foam.u2.TableView'
   ],
 
+  imports: [
+    'window'
+  ],
+
   constants: {
-    CSS_NAME: 'foam-u2-search-FilterController'
+    CSS_NAME: 'foam-u2-search-FilterController',
```

Comment:
Yes, that's correct according to my understanding. Let me check that my memory corresponds to reality... yes, it's badly broken without that.